pkg/kube_config_manager/kube_config_manager.go

```diff
@@ -148,7 +148,7 @@ func (kcm *kubeConfigManager) changeOrCreateKubeConfig(configChangeFunc func(*v1
 			return err
 		}
 
-		_, err := kcm.KubeClient.CoreV1().ConfigMaps(kcm.Namespace).Update(obj)
+		_, err := kcm.KubeClient.CoreV1().ConfigMaps(kcm.Namespace).Update(context.TODO(), obj, metav1.UpdateOptions{})
```

Comment:
I started thinking that maybe we should add background context with a timeout to all Kubernetes operations. We can leave it as-is for now, though.

Reply:
I would advise creating an issue regarding this comment.